extensions/quarkiverse-barcode-okapi.yaml

```diff
@@ -0,0 +1,6 @@
+---
+group-id: "io.quarkiverse.barcode"
+artifact-id: "quarkus-okapi"
```

Comment:
Shouldn't it be `quarkus-barcode-okapi`

Reply:
No i made all the libs just the name like it is above in case we ever wanted to separate out to its own separate extension i thought it would make it easier.